src/simulation/environment/albedo/_UnitTest/test_albedo.py

```diff
@@ -210,6 +210,43 @@ def unitAlbedo(show_plots, planetCase, modelType, useEclipse):
     print("This test uses a relative accuracy value of " + str(errTol * 100) + " percent")
 
     return [testFailCount, ''.join(testMessages)]
+
```

Comment:
Right now this test will always provide a BSK_ERROR messages when I run `pytest`.  To turn off this expected warning, please include
```
from Basilisk.architecture import bskLogging
```
and then, after creating the Albedo module, run 
```
albModule.bskLogger.setLogLevel(bskLogging.BSK_SILENT)`
```
This will silence this expected error message in this module.

Reply:
Be sure to rebase your branch on latest develop as well after making this change.  I just approved your other PR.

Reply:
Updated!